### DISCUSSION TITLE
Multi Camera Synchronization

### DISCUSSION THREAD

**Discussion by SoppierBrute9** [bonsai-rx/bonsai#1535] [Q&A]
Is there a way to synchronize three cameras, the tutorial only shows how to synchronize two cameras, but I need to do three. I want to start the workflow which will cause the three cameras to start recording and then stop when the workflow ends.

Also when I record on two of the cameras the video gets compressed into a one second video even though it was recorded for longer.

**Comment by bruno-f-cruz** [comment]
Have you tried the solution suggested in your previous post? https://github.com/bonsai-rx/bonsai/discussions/1522#discussioncomment-6745749

In the future please avoid duplicating posts and instead stick with the same discussion unless you have a different question. Regarding the length of the video, please include the workflow you are using as suggested in the discussions. Thanks!

**Reply by SoppierBrute9** [reply]
Sorry, the answer in the last post seemed more of a question on how I would want to sync the cameras. I tried using combinelatest and its seems to be working as I want it to.

**Reply by bruno-f-cruz** [reply]
Ok I will close this for now if you dont mind. IF you have more questions feel free to ask them there! thanks!

**Comment by SoppierBrute9** [comment]
Sorry, the answer in the last post seemed more of a question on how I would want to sync the cameras. I tried using combinelatest and its seems to be working as I want it to.
![IMG_2633](https://github.com/bonsai-rx/bonsai/assets/142339490/9495d350-f56e-4f5a-ab32-292fd67f0e78)
